airbyte-integrations/connectors/source-auth0/source_auth0/schemas/organization_member_roles.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": ["object", "null"],
+  "properties": {
+    "id": {
+      "type": ["string", "null"]
```

Comment:
Can you and additionalProperties and set it to true? https://docs.airbyte.com/connector-development/schema-reference/

Reply:
thank you @sajarin for noticing that, fixed!